sqlite.go

```diff
@@ -26,6 +26,7 @@ package sqlite
 // #cgo CFLAGS: -DSQLITE_ENABLE_PREUPDATE_HOOK
 // #cgo CFLAGS: -DSQLITE_USE_ALLOCA
 // #cgo CFLAGS: -DSQLITE_ENABLE_COLUMN_METADATA
+// #cgo windows LDFLAGS: -Wl,-Bstatic -lwinpthread -Wl,-Bdynamic
```

Comment:
This has a side effect of causing other CGO codes dynamically link their libraries because of the `-Bdynamic` flag, hence depending on DLLs (in a case we recently encountered it's libstdc++-6.dll). Even worse, the behavior can not be overridden via a `-static` or `-Bstatic` CGOLDFLAGS in the build script.

I think it's more of the application builder's responsibility to decide if the libraries need to be statically linked and a library can highlight it in its documents. Or if you prefer to enforce static linking of certain library, it would be better to enforce it rigidly and avoid other assumptions.

I would prefer to remove this line entirely and highlight in documents, but it's also okay to remove `-Wl,-Bdynamic` so the developers of applications and other libraries can freely have their own flags. How do you think @fasterthanlime @crawshaw ?

Reply:
disclaimer: I don't use this library anymore.

Linker flags are... unfortunate 😭  I didn't foresee that `-Wl,-Bdynamic` causing problems! Maybe [this can be used](https://stackoverflow.com/a/44164350/1021683) to *only* link winpthread statically?

We'd have:

```
// #cgo windows LDFLAGS: -l:libwinpthread.a
```

Reply:
No worries and either are fine with me! Thanks!